Cargo.toml

```diff
@@ -38,6 +38,7 @@ serde_json = "1.0.122"
 toml = "0.7.3"
 walkdir = "2.3"
 filetime = "0.2.9"
+indoc = "1.0"
```

Comment:
clippy_dev also has this as a dependency fwiw, but I can also drop it since it's only used to avoid having to have awkward indentation in one of the format strings